pkg/server/commands/batch_check_command_test.go

```diff
@@ -254,6 +263,58 @@ func TestBatchCheckCommand(t *testing.T) {
 		// response should have 1 key per check
 		require.Equal(t, len(response), numChecks)
 	})
+
+	t.Run("uses_command_cache_to_resolve_dupe_checks", func(t *testing.T) {
+		mockCheckResolver := graph.NewMockCheckResolver(mockController)
+		cmd := NewBatchCheckCommand(ds, mockCheckResolver, ts)
+		numChecks := 5
+		totalChecks := numChecks * 2
```

Comment:
**Suggestion:**
This is confusing IMO. `numChecks` is actually the number of duplicate batches.
```suggestion
		numChecks := 5
		totalChecks := numChecks * 2
```
Does this test need a loop at all? Would probably be easier to understand if you just hardcoded the payload to include a couple duplicates and one non-duplicate.